docs/resources/backup_config.md

```diff
@@ -0,0 +1,117 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "cockroach_backup_config Resource - terraform-provider-cockroach"
+subcategory: ""
+description: |-
+  The backup configuration for a cluster.
+  Each cluster has a backup configuration that determines if backups are enabled,
+  how frequently they are taken and how often they are retained for.  This
+  resource allows configuring those settings.  It's important to note that the
+  existence or lack of this resource doesn't not indicate whether a backup
+  configuration exists or not. Rather, if Terraform is managing it or not.
+  As a result of this, removal of the resource from the Terraform configuration
+  will leave the backup configuration in the state it was before the removal.
```

Comment:
Does this mean, if you remove the backup_config tf resource, it just won't have any affect on the the configuration at all? If I have understood that correctly, perhaps it could be something like:

> As a result, removing the resource from the Terraform configuration will not affect the settings in the backup configuration.

Reply:
yes that's right. The suggestion sounds good to me.